docs/content/misc/faq.ngdoc

```diff
@@ -174,8 +174,8 @@ Yes, AngularJS can use [jQuery](http://jquery.com/) if it's present in your app
 application is being bootstrapped. If jQuery is not present in your script path, AngularJS falls back
 to its own implementation of the subset of jQuery that we call {@link angular.element  jQLite}.
 
-AngularJS 1.3 only supports jQuery 2.1 or above. jQuery 1.7 and newer might work correctly with AngularJS
-but we don't guarantee that.
+AngularJS 1.8 only supports jQuery 3.5.1 or above. Earlier versions of jQuery might work correctly
```

Comment:
That'd be very restrictive... Can't we maintain existing support?

Reply:
I didn't want to suggest that people use a version of jQuery that might have a security flaw...
What would be better wording?

Reply:
How about:

> For AngularJS 1.8 we suggest jQuery 3.5.1 or above. Earlier versions of jQuery might work correctly with AngularJS but we don't guarantee that.

Reply:
Actually after reading the other comments I am going with:

```
For AngularJS 1.8 we support jQuery 2.1+ but we suggest jQuery 3.5.1 or above to avoid a potential
security issue. Earlier versions of jQuery might work correctly with AngularJS but we don't guarantee
that.
```

Reply:
Yeah, even before these vulnerabilities fixed in jQuery 3.5.0 jQuery 2.1/2.2 already had a few reported security issues that were only fixed in jQuery 3.0.0. And yet we didn't drop support for them.

Pointing out we recommend 3.5.1+ is fine.